cgpsa

```diff
@@ -265,12 +267,17 @@ sub initialize
   if ($root_conf->{no_configuration})
   {
     # Exit if there's no configuration, since we can't do anything useful.
-    
+
     error_exit("Unable to read CGPSA configuration file, exiting.");
   }
+  if (!$root_conf->{x_junk_score_thresholds}) {
+    # Warn if the root configuration specifies invalid junk score thresholds.
+
+    print("* Invalid X-Junk-Score thresholds in configuration, X-Junk-Score disabled.")
```

Comment:
Below, debug(1, ...) is used

Reply:
> whether I should disable the feature entirely (in a non-overridable-by-domain-or-user-conf way) if it's misconfigured in this way at the root level

We are only using the root level configuration and do not intend to use it differently. Therefore, it's difficult to tell. However, it sounds reasonable what you are describing.

Reply:
Given that this is a CGPSA helper startup message, I'm going to leave it as-is.